wazimap_ng/datasets/migrations/0128_auto_20220711_0723.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.28 on 2022-07-11 07:23
+
+import django.core.validators
+from django.db import migrations, models
+import wazimap_ng.datasets.models.upload
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('datasets', '0127_auto_20220401_0949'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='datasetfile',
+            name='document',
+            field=models.FileField(help_text='\n            Uploaded document should be less than 3000.0 MiB in size and\n            file extensions should be one of xls, xlsx, csv.\n        ', upload_to=wazimap_ng.datasets.models.upload.get_file_path, validators=[django.core.validators.FileExtensionValidator(allowed_extensions=['xls', 'xlsx', 'csv']), wazimap_ng.datasets.models.upload.file_size]),
+        ),
+        migrations.AlterField(
+            model_name='historicaldatasetfile',
+            name='document',
+            field=models.TextField(help_text='\n            Uploaded document should be less than 3000.0 MiB in size and\n            file extensions should be one of xls, xlsx, csv.\n        ', max_length=100, validators=[django.core.validators.FileExtensionValidator(allowed_extensions=['xls', 'xlsx', 'csv']), wazimap_ng.datasets.models.upload.file_size]),
+        ),
+    ]
```

Comment:
I don't understand why this migration happened. Perhaps there was some old change that was not included in a migration

But the change looks fine